app/src/test/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdateTest.kt

```diff
@@ -169,7 +169,7 @@ class CustomDrugEntryUpdateTest {
   }
 
   @Test
-  fun `when the drug is fetched and is not deleted, then update the model and set frequency`() {
+  fun `when the drug is fetched and is not deleted, then update the model, set frequency and update the progress state to done`() {
```

Comment:
Just an FYI, we can also just call it as update model so that we don't have to add everything to the test name.

Reply:
Here the different things that are added are the effects that are being dispatched. Should those be abstracted under `update the model` as well?